2022-01-11 11:00:54,261 INFO    MainThread:60356 [wandb_setup.py:_flush():71] setting env: {}
2022-01-11 11:00:54,261 INFO    MainThread:60356 [wandb_setup.py:_flush():71] setting login settings: {'anonymous': 'must'}
2022-01-11 11:00:54,262 INFO    MainThread:60356 [wandb_init.py:_log_setup():371] Logging user logs to E:\Pytorch-UNet\wandb\run-20220111_110054-2syg1xeu\logs\debug.log
2022-01-11 11:00:54,262 INFO    MainThread:60356 [wandb_init.py:_log_setup():372] Logging internal logs to E:\Pytorch-UNet\wandb\run-20220111_110054-2syg1xeu\logs\debug-internal.log
2022-01-11 11:00:54,263 INFO    MainThread:60356 [wandb_init.py:init():404] calling init triggers
2022-01-11 11:00:54,263 INFO    MainThread:60356 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-01-11 11:00:54,265 INFO    MainThread:60356 [wandb_init.py:init():449] starting backend
2022-01-11 11:00:54,265 INFO    MainThread:60356 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2022-01-11 11:00:54,278 INFO    MainThread:60356 [backend.py:ensure_launched():199] starting backend process...
2022-01-11 11:00:55,284 INFO    MainThread:60356 [backend.py:ensure_launched():205] started backend process with pid: 65736
2022-01-11 11:00:55,285 INFO    MainThread:60356 [wandb_init.py:init():458] backend started and connected
2022-01-11 11:00:55,289 INFO    MainThread:60356 [wandb_init.py:init():519] updated telemetry
2022-01-11 11:00:55,345 INFO    MainThread:60356 [wandb_init.py:init():549] communicating current version
2022-01-11 11:00:56,459 INFO    MainThread:60356 [wandb_init.py:init():554] got version response 
2022-01-11 11:00:56,459 INFO    MainThread:60356 [wandb_init.py:init():564] communicating run to backend with 30 second timeout
2022-01-11 11:01:26,460 WARNING MainThread:60356 [wandb_init.py:init():938] interrupted
Traceback (most recent call last):
  File "C:\Users\2110012.WXCNL-5887\anaconda3\envs\Pytorch-UNet\lib\site-packages\wandb\sdk\wandb_init.py", line 917, in init
    run = wi.init()
  File "C:\Users\2110012.WXCNL-5887\anaconda3\envs\Pytorch-UNet\lib\site-packages\wandb\sdk\wandb_init.py", line 565, in init
    ret = backend.interface.communicate_run(run, timeout=30)
  File "C:\Users\2110012.WXCNL-5887\anaconda3\envs\Pytorch-UNet\lib\site-packages\wandb\sdk\interface\interface.py", line 205, in communicate_run
    return self._communicate_run(run, timeout=timeout)
  File "C:\Users\2110012.WXCNL-5887\anaconda3\envs\Pytorch-UNet\lib\site-packages\wandb\sdk\interface\interface_queue.py", line 334, in _communicate_run
    resp = self._communicate(req, timeout=timeout)
  File "C:\Users\2110012.WXCNL-5887\anaconda3\envs\Pytorch-UNet\lib\site-packages\wandb\sdk\interface\interface_queue.py", line 232, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "C:\Users\2110012.WXCNL-5887\anaconda3\envs\Pytorch-UNet\lib\site-packages\wandb\sdk\interface\router.py", line 32, in get
    is_set = self._object_ready.wait(timeout)
  File "C:\Users\2110012.WXCNL-5887\anaconda3\envs\Pytorch-UNet\lib\threading.py", line 551, in wait
    signaled = self._cond.wait(timeout)
  File "C:\Users\2110012.WXCNL-5887\anaconda3\envs\Pytorch-UNet\lib\threading.py", line 299, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
